code/drasil-lang/Language/Drasil/Chunk/Concept/Core.hs

```diff
@@ -18,9 +18,10 @@ data DefnAndDomain = DAD { _defn' :: Sentence, _cdom' :: [UID]}
 makeLenses ''DefnAndDomain
 
 -- | The ConceptChunk datatype is a Concept
--- ConDict is not exported, nor are _idea, and _dad.
+-- ConDict is not exported, nor are _idea, _dad, and _sn.
 data ConceptChunk = ConDict { _idea :: IdeaDict
                             , _dad :: DefnAndDomain
+                            , _sn :: Maybe ShortName
```

Comment:
No sure if this is quite right - please ask say @niazim3 if this corresponds to the current code.

Reply:
From skimming #695's comments, it looks like the `Maybe ShortName` was removed so @Mornix can look into it, which I believe he worked on in #745.

This `Maybe ShortName` field should not be added.

Reply:
Core.hs has been updated to match Master's in d1fb4384458fa7462ddad0ab0163334ca28c5e7e. Now this file is exactly same as Master's.